src/core/currency/wallet/currency-wallet-export.js

```diff
@@ -33,13 +33,31 @@ export function searchStringFilter(
   let out = txs
   if (searchString != null && searchString !== '') {
     const lowerCaseSearchString = searchString.toLowerCase()
+    const noSeparatorsSearchString = searchString
+      .replace('.', '')
+      .replace(',', '')
     out = out.filter(tx => {
+      if (!/[A-Za-z]/.test(searchString)) {
+        if (tx.nativeAmount.indexOf(noSeparatorsSearchString) >= 0) return true
+      }
       if (tx.metadata != null) {
-        const { category = '', name = '', notes = '' } = tx.metadata
+        const {
+          category = '',
+          name = '',
+          notes = '',
+          exchangeAmount = {}
+        } = tx.metadata
         if (
           category.toLowerCase().indexOf(lowerCaseSearchString) >= 0 ||
           name.toLowerCase().indexOf(lowerCaseSearchString) >= 0 ||
-          notes.toLowerCase().indexOf(lowerCaseSearchString) >= 0
+          notes.toLowerCase().indexOf(lowerCaseSearchString) >= 0 ||
+          (tx.wallet != null &&
+            exchangeAmount[tx.wallet.fiatCurrencyCode] &&
+            exchangeAmount[tx.wallet.fiatCurrencyCode]
+              .toString()
```

Comment:
Believe the exchange amount is already a string. Please double check and if so remove the toString

Reply:
exchangeAmount in EdgeMetaData is a number 

exchangeAmount?: { [fiatCurrencyCode: string]: number }